dff/core/engine/conditions.py

```diff
@@ -201,7 +200,11 @@ def has_last_labels_condition_handler(ctx: Context, actor: Actor, *args, **kwarg
 
 @validate_arguments
 def true(*args, **kwargs) -> Callable:
```

Comment:
Should we add Callable signature (☝️): `Callable[[Context, Actor, Any, Any], bool]` (AFAIK) here.